### ISSUE TITLE
Visual blank

### ISSUE THREAD

**Issue by shixinsong** [JiawangBian/SC-SfMLearner-Release#51]
When I execute the sh scripts/run_inference.sh command, a depth map is generated under the result file, but these pictures are blank pictures. The following is the model I trained and the problems that occurred. 
Hoping to get your help,
Thanks.
![Screenshot from 2020-06-23 11-01-24](https://user-images.githubusercontent.com/67302433/85356760-4fd43e00-b542-11ea-8544-7dfee75dbad6.png)
![LS5CJ_D{M@29BC569KL3ZTP](https://user-images.githubusercontent.com/67302433/85356766-5498f200-b542-11ea-9711-4b7f8c604b5a.png)

**Comment by JiawangBian** [comment]
Can you give more information on your training e.g., which script you use, and the visualization of training loss for debug?

**Comment by shixinsong** [comment]
Hi,
I retrained the following and found the problem solved! In order to accelerate training before, epoch chose 10.
Thanks.

**Comment by XiangYu610** [comment]
Hello, may I ask if the appearance of this visually blank image is related to the number of training epochs? In theory, as long as the model is trained, the input image can output a depth map, but the image quality and performance may differ.
